### PR TITLE
Feature/get lis posts

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In thiss PR, I update view get list post

### CODE REVIEW

File: property_web/posts/templates/list_posts.html [hunk 1]
```diff
@@ -2,19 +2,17 @@
...
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/list_post.css' %}">
 
     <h2 class="subheading">Hot Posts</h2>
     <ul class="post-list">
         {% for post in hot_posts %}
             <li class="post-item">
                 <a href="{% url 'post_detail' post_id=post.id %}">
-                    <img class="post-image-ava" src="{{ post.image_set.first.image.url }}" alt="{{ post.title }} Image">
+                    {% if post.image %}
```
Comment (cuongdoan-agilityio) [inline property_web/posts/templates/list_posts.html:13]: Do we have a default image if not post's image ?

File: property_web/posts/views.py [hunk 2]
```diff
@@ -42,13 +43,19 @@ def post_list(request):
     """
     View function to display a list of posts categorized as hot and normal.
     """
+    posts = Post.objects.filter(status='AVAILABLE').values(
+        'id', 'title', 'price', 'address', 'hot_post'
+    )
```
Comment (cuongdoan-agilityio) [inline property_web/posts/views.py:48]: Why don't you sort by creation time?

File: property_web/posts/views.py [hunk 3]
```diff
@@ -42,13 +43,19 @@ def post_list(request):
...
+
+    for post in posts:
+        image = Image.objects.filter(post_id=post['id']).first()
+        if image:
+            post['image'] = image.image.url
+        else:
+            post['image'] = None
```
Comment (cuongdoan-agilityio) [inline property_web/posts/views.py:55]: Please format this code.

File: property_web/posts/views.py [hunk 4]
```diff
@@ -42,13 +43,19 @@ def post_list(request):
...
 
-    hot_posts = Post.objects.filter(
-        hot_post=True, status='AVAILABLE'
-    ).order_by("-created_at")
-    normal_posts = Post.objects.filter(
-        hot_post=False, status='AVAILABLE'
-    ).order_by("-created_at")
+    hot_posts = [post for post in posts if post['hot_post']]
+    normal_posts = [post for post in posts if not post['hot_post']]
```
Comment (cuongdoan-agilityio) [inline property_web/posts/views.py:58]: Please format this code.

File: property_web/posts/views.py (hunk 2 above)
Comment (hothanhtuan1209) [reply property_web/posts/views.py:48]: Done

**Comment by hothanhtuan1209** [comment]
image
image
image
image

**Pull request merged**